azure-pipelines.yml

```diff
@@ -0,0 +1,119 @@
+trigger:
+  branches:
+    include:
+    - refs/tags/v*
+jobs:
+- job: Build
+  pool:
+    name: Azure-IoT-EdgeExperience-1ES-Hosted-Linux
+    demands:
+      - ImageOverride -equals MMSUbuntu18.04TLS
+
+  steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '14.x'
+        checkLatest: true
+
+    - script: |
+          npm install yo
+          npm install 
+      displayName: 'npm install'
+      
+    - script: |
+          npm run lint 
+      displayName: 'npm lint'
+
+    - task: Npm@1
+      displayName: "npm Packaging"
+      inputs:
+          command: 'custom'
+          customCommand: 'pack'
+
+    - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+      displayName: 'SBOM Generation Task'
+      inputs:
+        BuildDropPath: '$(Build.ArtifactStagingDirectory)'
+
+    - task: CopyFiles@2
+      inputs:
+        SourceFolder: '$(Build.SourcesDirectory)'
+        Contents: '*.tgz'
+        TargetFolder: '$(Build.ArtifactStagingDirectory)/Package'
+    - task: PublishBuildArtifacts@1
+      inputs:
+        PathtoPublish: '$(Build.ArtifactStagingDirectory)/Package'
+        ArtifactName: 'drop'
+        publishLocation: 'Container'
+
+    - task: Npm@1
+      displayName: "npm Test"
+      inputs:
+        command: 'custom'
+        customCommand: 'test'
+
+    - task: Bash@3
+      inputs:
+        targetType: 'inline'
+        script: |
+          if [[ $(Build.SourceBranch) =~ ^refs/tags/v?[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+          echo "##vso[task.setvariable variable=NPM_PUBLISH_FLAG;]true"
+          fi
```

Comment:
Is this step needed in this job? The env variable is only used in UploadtoStorage job.

Reply:
deleted the task